graph-commons/src/test/scala/ch/datascience/generators/CommonGraphGenerators.scala

```diff
@@ -29,7 +29,7 @@ import ch.datascience.crypto.AesCrypto
 import ch.datascience.generators.Generators._
 import ch.datascience.graph.config.RenkuBaseUrl
 import ch.datascience.graph.model.SchemaVersion
-import ch.datascience.graph.model.users.{Email, Name, Username}
+import ch.datascience.graph.model.users.{Affiliation, Email, Name, Username}
```

Comment:
If you are trying to be complete, there is also `schema:alternateName` afaik

Reply:
not sure if it's currently used anywhere, but I think zenodo was including it in some dataset metadata and that's why we added it.